libbeat/processors/add_cloud_metadata/provider_alibaba_cloud.go

```diff
@@ -0,0 +1,41 @@
+package add_cloud_metadata
+
+import "github.com/elastic/beats/libbeat/common"
+
+// Alibaba Cloud Metadata Service
+// Document https://help.aliyun.com/knowledge_detail/49122.html
+func newAlibabaCloudMetadataFetcher(c common.Config) (*metadataFetcher, error) {
+	ecsMetadataHost := "100.100.100.200"
```

Comment:
Yyep, maybe metadata is not a really important feature for Chinese users, the design looks not so serious in both Tencent and Alibaba provider :-)

> Couldn't make much sense of the docs above unfortunately ;-)
You mean remove the document comment?